cheaty@centurix/files/cheaty@centurix/applet.js

```diff
@@ -43,7 +50,7 @@ SheetMenuItem.prototype =
 		this.addActor(this.icon);
 		this.icon.realize();
 
-		this._tooltip = new Tooltips.Tooltip(this.actor, sheet.name + ' (version ' + sheet.version + ')\n' + sheet.description + '\nAuthor: ' + sheet.author + '\nEmail: ' + sheet.email + '\nRepository: ' + sheet.repository);
+		this._tooltip = new Tooltips.Tooltip(this.actor, sheet.name + " " + _("version") + " " + sheet.version + "\n" + sheet.description + _("\nAuthor:") + " " + sheet.author + _("\nEmail:") + " " + sheet.email + _("\nRepository:") + " " + sheet.repository);
```

Comment:
"\n" + _("Author")

etc. Do not start/end with whitespaces/newline, if possible